lib/dom.js

```diff
@@ -751,6 +752,20 @@ declare class HTMLFormElement extends HTMLElement {
     submit(): void;
 }
 
+declare class HTMLIFrameElement extends HTMLElement {
+    allowFullScreen: boolean;
+    frameBorder: number;
+    height: number;
+    marginHeight: number;
+    marginWidth: number;
+    name: string;
+    scrolling: string;
+    sandbox: string;
```

Comment:
Looks like this: https://dom.spec.whatwg.org/#interface-domtokenlist

It's basically a space-separated list of strings (tokens). Should I add `DOMTokenList` as a definition and use it instead?